recipes/enjincppsdk/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(KEEP_RPATH)
```

Better compatibility on Mac

---

Just for clarification, is it `KEEP_RPATH` or `KEEP_RPATHS`?

---

Ops, it was a typo indeed, it's `KEEP_RPATHS` https://docs.conan.io/en/latest/reference/generators/cmake.html#conan-basic-setup